app/gui/src/ide/initializer.rs

```diff
@@ -94,26 +78,36 @@ impl Initializer {
         //      issues to user, such information should be properly passed
         //      in case of setup failure.
 
+        match self.initialize_ide_controller_with_retries().await {
+            Ok(controller) => {
+                let ide = Ide::new(ensogl_app, view.clone_ref(), controller);
+                if let Some(project) = &self.config.project_to_open {
+                    ide.open_or_create_project(project.clone());
+                }
+                info!("Setup done.");
```

Comment:
This log does not tell much. Seeing "setup done" in terminal I would not understand what it is about. I know it is from old code, but should be fixed here I think.